### ISSUE TITLE
Building from source - Windows...

### ISSUE THREAD

**Issue by rorywalsh** [Tracktion/pluginval#45]
One of my plugins causes PluginVal to crash when the strictness level of ramped up past 6. I would like to look a little deeper but the dev version of PluginVal performs even worse. It throws up an exception for every VST3 plugin I try to scan:

![image](https://user-images.githubusercontent.com/2689817/118456078-1fb10300-b6f1-11eb-9c30-7e4ddd786d87.png)

Maybe I should jump to the master branch?

**Comment by drowaudio** [comment]
That was a change added in the last commit. Can you please post the full stack trace so I can see what's triggering it please?

Yes, using the master branch for now should work. Sorry about that.

**Comment by rorywalsh** [comment]
This any good to you:

```
        pluginval.exe!juce::VariantConverter<std::vector<double,std::allocator<double>>>::fromVar(const juce::var & v) Line 35	C++
 	pluginval.exe!ValidatorChildProcess::processRequest(juce::MemoryBlock mb) Line 484	C++
 	pluginval.exe!ValidatorChildProcess::processRequests() Line 463	C++
 	pluginval.exe!ValidatorChildProcess::run() Line 432	C++
 	pluginval.exe!juce::Thread::threadEntryPoint() Line 96	C++
 	pluginval.exe!juce::juce_threadEntryPoint(void * userData) Line 119	C++
 	pluginval.exe!juce::threadEntryProc(void * userData) Line 62	C++
 	pluginval.exe!thread_start<unsigned int (__cdecl*)(void *),1>(void * const parameter) Line 97	C++
```

**Comment by drowaudio** [comment]
I think this is fixed in the 1.0.0 release
